Project.toml

```diff
@@ -1,47 +1,27 @@
 name = "ScikitLearn"
 uuid = "3646fa90-6ef7-5e7e-9f22-8aca16db6324"
-version = "0.6.6"
+version = "0.7.0"
 
 [deps]
 Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
-Conda = "8f4d0f93-b110-5947-807f-2305c1781a2d"
-DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
-Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
-IterTools = "c8e1da08-722c-5040-9ed9-7db0dc04731e"
-LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
-Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
-Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
-PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
-Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+PythonCall = "6099a3de-0909-46bc-b1f4-468b9a2dfc0d"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
 ScikitLearnBase = "6e75b9c4-186b-50bd-896f-2d2496a4843e"
-SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
-StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
-VersionParsing = "81def892-9a0e-5fdd-b105-ffc91e053289"
+Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
-Compat = "2.2, 3, 4"
-Conda = "1.5.2, 1.6, 1.7"
-DataFrames = "0.20, 0.21, 0.22, 1"
-IterTools = "1.2, 1.3"
+Compat = "3, 4"
```

Comment:
I suppose inclusion of Compat is a very temporary measure?

Reply:
Removed it!